DOCKER/docker-compose

```diff
@@ -0,0 +1,34 @@
+version: "3.8"
+
+services:
+
+  postgis:
+    build:
+      context: .
+      dockerfile: dockerfile-toms-test-postgis
+    hostname: toms_postgis2
+    environment:
+      - POSTGRES_PASSWORD=postgis
+      - POSTGRES_DBNAME=gis
+    volumes:
+      - "pg_data_toms_test:/var/lib/postgresql"
+      - /home/QGIS/plugins/TOMs/DATAMODEL:/docker-entrypoint-initdb.d
+      - /home/QGIS/plugins/TOMs/test/data:/docker-entrypoint-initdb.d
```

Comment:
Does this work ? Not sure it's possible/supported to mount two directories to the same mount point ?

Also, I'm not putting the datamodel migrations files directly in docker-entrypoint-initdb.d will work. For example, currently, you're supposed to run migrations 0001 first, then import the test data, then run the rest of the migrations, as the test-data is adapted to version 0001.
It would be better to put the sqls elsewhere (e.g. `/io/datamodel` and `/io/testdata`, and to add a bash script (bash) in `docker-entrypoint-initdb.d` that calls these files, so you can control the order in which each migration is run.

And you should use relative paths in docker-compose (paths are relative to the compose file), so it works for everybody.

```suggestion
      - ../DATAMODEL:/io/datamodel
      - ../TOMs/test/data:/io/testdata
```

Reply:
The relative path is now included. It is assumed that docker-compose up is issued from the DOCKER folder of the plugin. Is this sensible?

Reply:
Yes looks good